src/plays/react-gradients/Readme.md

```diff
@@ -0,0 +1,75 @@
+# React Gradients
+
```

Comment:
@ammaaraslam I have not seen any play creator has made the readme so extensive and detailed. Thanks a lot. It will help many.